TrainingHW4/TrainingHW4/Program.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+
+namespace TrainingHW4
+{
+    class Program
+    {
+        static int FindMinValue(int[] array)
+        {
+            int Min = array[0];
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                if (array[i] < Min)
+                {
+                    Min = array[i];
+                }
+
+            }
+
+
+            return Min;
+        }
+
+        static int FindMaxValue(int[] array)
+        {
+            int Max = array[0];
+
+
+            for (int i = 0; i < array.Length; i++)
+            {
+
+                if (array[i] > Max)
+                {
+                    Max = array[i];
+                }
+
+            }
+
+            return Max;
+
+        }
+
+
+        static void ChangeMax_and_Min(int[] array)
+        {
+            int Max = array[0];
+            int Min = array[0];
+
```

Comment:
Read this [article](https://docs.microsoft.com/en-us/dotnet/api/system.array.indexof?view=net-6.0) and tell me how we can simplify this function.

Reply:
we can use index to change min and max values

Reply:
i have a question))
i can change min and max value using index if i have only one max and min value
so I've chosen this way if there are more than one min/max values in array